controllers/task.go

```diff
@@ -98,6 +94,31 @@ func Tasks(ctx context.Context, w http.ResponseWriter, r *http.Request) (status
 		return http.StatusUnauthorized, nil
 	}
 
+	if r.Method == "GET" {
```

Comment:
I think it would look better with :
```golang
switch r.Method{
    case "GET":
        return getAllTasks()
    case "POST":
        return post()
    default:
        return 401, nil

```